gradle.properties

```diff
@@ -1,6 +1,6 @@
 org.gradle.daemon=false
 
-elasticsearchVersion=8.13.0
+elasticsearchVersion=8.14.0
```

Comment:
This file must not be in this PR. It would set the 8.13 branch to the wrong version.

Reply:
yikes, I should have looked into why Jan's commit showed up here. My bad.